### ISSUE TITLE
Docker example broken

### ISSUE THREAD

**Issue by maparham** [eclipse-arrowhead/core-java-spring#430]
Here are the issues I'm facing when trying to get a basic demo run:
1. The instructions for running a demo are hard to follow and should be written in a clear sequence of steps (copy-paste ready)
2.  I copied `create_empty_arrowhead_db.sql` to `docker/` but the script does not work.
3. As a result, running the docker-compose*.yml prints numerous "access denied" errors like the following snippet.

`
mysql            | 2023-05-17 12:33:19 4 [Warning] Access denied for user 'authorization'@'192.168.224.5' (using password: YES)
mysql            | 2023-05-17 12:33:20 3 [Warning] Access denied for user 'service_registry'@'192.168.224.4' (using password: YES)
authorization    | 2023-05-17 12:33:21.259 ERROR c2f114373bd9 --- [cLoggerConfig-4] c.z.h.p.HikariPool                       : HikariPool-1 - Exception during pool initialization.
authorization    | java.sql.SQLException: Access denied for user 'authorization'@'192.168.224.5' (using password: YES)
`

How to NOT write instructions:
_Don't forget to create...
Don't forget to copy..._

**Comment by maparham** [comment]
I closed the issue as I don't think this will ever be addressed...moving on to other options.

**Comment by borditamas** [comment]
Hello,

we have just released the v4.6.1, where a more flexible docker approach has been taken.
[Take a look](https://github.com/eclipse-arrowhead/core-java-spring/wiki/Docker) and please open a new issue if you still runs into some error.
